.gitignore

```diff
@@ -37,11 +37,13 @@ test/binaries
 # coverage artifacts
 .coverprofile
 gover.coverprofile
+vendor/
 
 e2e-tests
 coverage.txt
 test/e2e/e2e\.test
 .env
+.envrc
```

Comment:
Was it accidentally added here? do we need to ignore it？

Reply:
👌